packages/cli/src/commands/sandbox/sandbox-seed/sandbox_seed_command.ts

```diff
@@ -35,15 +36,22 @@ export class SandboxSeedCommand implements CommandModule<object> {
    * @inheritDoc
    */
   handler = async (): Promise<void> => {
+    printer.startSpinner('');
```

Comment:
Can we do this to make it more descriptive?
```suggestion
    printer.startSpinner('Running seed...');
```